tests/reconnect_test.cpp

```diff
@@ -31,17 +31,9 @@ namespace tests {
 static size_t backends_count = 1;
 static int stall_count = 2;
 static int wait_timeout = 1;
-/* Check timeout must be at least wait_timeout * stall_count seconds to
- * guarantee that reconnection process will not affect test_failed_connection_restore().
- * Otherwise reconnection thread may send transactions when command sending is already
- * disabled, while test logic is not even started, so stall counter can reach its limit
- * and BACKEND_STATUS will be send. These circumstances lead to network state destruction,
- * thereby requests from test_failed_connection_restore() will return ENXIO error,
- * instead of expected ETIMEDOUT error.
- */
-static int check_timeout = wait_timeout * stall_count;
+static int reconnect_batch_size = 2;
```

Comment:
Why is it an `int` if it originally is `uint8_t`?

Reply:
Fixed.